tools/sof-kernel-log-check.sh

```diff
@@ -312,11 +316,10 @@ ignore_str="$ignore_str"'|elan_i2c i2c-ELAN0000:.*: invalid report id data'
 ignore_str="$ignore_str"'|rt700 sdw:.:.*:700:.: Bus clash detected'
 
 # confirm begin_timestamp is in UNIX timestamp format, otherwise search full log
-journalctlflag="-k -q --no-pager --utc --output=short-monotonic --no-hostname"
 if [[ $begin_timestamp =~ ^[0-9]{10} ]]; then
-    cmd="journalctl $journalctlflag --since=@$begin_timestamp"
+    cmd="journalctl_cmd --since=@$begin_timestamp"
 else
-    cmd="journalctl $journalctlflag"
+    cmd="journalctl_cmd"
 fi
 
 declare -p cmd
```

Comment:
maybe it is also time to remove the debug log here.

Reply:
#529 is less than one week old so I would rather wait a bit longer.